eng/common/README.md

```diff
@@ -20,4 +20,6 @@ The 'Sync eng/common directory' PRs will be created in the language repositories
 3. More changes pushed to the **Tools PR**, will automatically triggered new pipeline runs in the respective **Sync PRs**. The **Sync PRs** are used to make sure the changes would not break any of the connected pipelines.
 4. Once satisfied with the changes;
     - First merge all the **Sync PRs**. The **Tools PR** contains links to all the **Sync PRs**
-    - Finally merge the **Tools PR**. Each **Sync PR** contains the link to the corresponding **Tools PR**. 
\ No newline at end of file
+    - Finally merge the **Tools PR**. Each **Sync PR** contains the link to the corresponding **Tools PR**. 
+
+See [detailed eng/common Workflow](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/281/Engineering-Systems-Tools?anchor=eng%2Fcommon-workflow)
```

Comment:
While folks may not be able to see the pipeline I think it is still worth including most of the steps in this file.

Reply:
I don't like that we are adding a protected link here as we will have to add this to our ignore-links.txt file in every repo which I'd like to avoid. Lets start by not having this link and get as much detail as we can directly in the MD file here. If we need more details or images we can add them here later.